README.md

```diff
@@ -1,3 +1,57 @@
 # scalaz-actors
 
-[![Gitter](https://badges.gitter.im/scalaz/scalaz-actors.svg)](https://gitter.im/scalaz/scalaz-actors?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
\ No newline at end of file
+[![Gitter](https://badges.gitter.im/scalaz/scalaz-actors.svg)](https://gitter.im/scalaz/scalaz-actors?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+# Summary / Goal
+High-performance, purely-functional library for building, composing, and supervising typed actors based on Scalaz ZIO.
+
+# Introduction / Highlights
+The [Actor Model](https://en.wikipedia.org/wiki/Actor_model) is used to build distributed highly scaleable applications. The core concept behind the actor model is the ability to create multiple actors which run concurrently. The actor would receive a message do some computation on the message and then output a new message. Each actor runs independently of each other with no shared state between them and as such failure of one actor won't have an affect on the running of another.
+
+In its simplest form the goal of this project is to provide the ability to write actors in Functional Way that are typed leveraging [scalaz-zio](https://github.com/scalaz/scalaz-zio). However for a fully usable system will need the following components:
+
+- Supervision
+
+- Actor Mailbox/Queue
+
+- Actor Monitoring and Metrics
+
+- Actor persistence
+
+- Message Stashing
+
+- Message Routing / Actor Registry
+
+This summarizes the goals of this project well. We will be adding a detailed
+technical design document explaining everything in more details here on the
+project wiki: [Technical Design Document](https://github.com/scalaz/scalaz-actors/wiki/Design)
+
+
+# Competition
```

Comment:
Under competition, it will be good to compare the competitors against the key selling points of the library—what drives a person to use Scalaz Actors, instead of Akka, for example?